packages/app/android/app/build.gradle

```diff
@@ -66,9 +79,11 @@ android {
             manifestPlaceholders = [applicationLabel: "Meli (Profile)"]
         }
         release {
-            // TODO: Add your own signing config for the release build.
-            // Signing with the debug keys for now, so `flutter run --release` works.
-            signingConfig signingConfigs.debug
+            // We do not intend to upload the .apk to any "App Store" but still
+            // sign releases with our developer keys to authenticate the
+            // official builds.
+            // https://docs.flutter.dev/deployment/android#configure-signing-in-gradle
+            signingConfig signingConfigs.release
```

Comment:
I thought it's cool to sign the .apks even if it's not for uploading it somewhere

Reply:
The build process also generates checksums, that could also be enough to authenticate them